settings.php

```diff
@@ -48,6 +48,26 @@
     );
     $settings->add(new admin_setting_configselect($name, $title, $description, $default, $choices));
 
+    /* Toggle display block choices */
+    $name = 'format_topcoll/defaultdisplayblocks';
+    $title = get_string('defaultdisplayblocks', 'format_topcoll');
+    $description = get_string('defaultdisplayblocks_desc', 'format_topcoll');
+    $default = array('search_forums','news_items','calendar_upcoming','recent_activity');
+    $choices = array('search_forums'=>'Search forums','news_items'=>'Latest announcements',
```

Comment:
This needs to be a list of installed blocks from the block manager with 'get_strings' on the other end, i.e get_string('pluginname', 'block_recent_activity')

Reply:
i.e. as stated before:

Use an associative array -> array(
'search_forums' => get_string(...),
'news_items' => get_string(...),
'calendar_upcoming' => get_string(...),
'recent_activity' => get_string(...)
)

but the keys will be from the list of installed blocks.

Reply:
i.e. see what core_plugin_manager::instance()->get_enabled_plugins('block') returns.

Reply:
or if that has agro, then core_plugin_manager::instance()->get_plugins_of_type('block').